### ISSUE TITLE
Feature Request: Subtypes of Webpage => AUP, TOS, Privacy, Shipping & Refund

### ISSUE THREAD

**Issue by inetbiz** [schemaorg/schemaorg#1308]
Give search AI real data they may check for user experience. All of these are important to eCommerce and Web Privacy & Security. The presence of these subtypes could assist AI develop scoring for a domain.

**Comment by Malvoz** [comment]
I agree there should be some type of PrivacyPolicyPage. There is already a TOS **property** (http://schema.org/termsOfService) which can be used with the types @Service and @WebAPI. There is also the property publishingPrinciples (http://schema.org/publishingPrinciples).

**Comment by jdevalk** [comment]
I'd like to raise this again, seems like PrivacyPolicyPage is a no brainer.

**Comment by jvandriel** [comment]
Is there really a need for new subtypes of Webpage? Seems like a possible pitfall of endless options to me.

Personally I've been using a combination of a multi typed entity for this (Article + some more specific external entity) as a ```mainEntityOfPage``` of a ```AboutPage```, eg:

```
<script type="application/ld+json">
{
  "@context":"https://schema.org",
  "@type":["Article","http://www.wikidata.org/entity/Q1999831"],
  "name":"Our privacy policy",
  "mainEntityOfPage":
  {
    "@type":"AboutPage",
    "@id":"http://example.com/privacy-policy"
  }
}
</script>
```
Q1999831 is this case refers to the Wikidata item for Privacy policy, which could also have been [Terms of service](http://www.wikidata.org/entity/Q7214489), or any other such 'term'.

Doesn't this suffice?

**Comment by danbri** [comment]
Is anyone here expecting to build something consuming such data?

**Comment by deerheart27** [comment]
Greetings! I would like to add to the conversation about termsOfService. I am new here, so please excuse me if this is not the right place to share feedback, example of implementations, suggestions, etc.

I just implemented [termsOfService](https://schema.org/termsOfService). The schema markup I used was:

`<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "WebPage",
  "url": "https://www.jennyelizabeth.com/terms-and-conditions",
  "name": "Terms and Conditions | Jenny Elizabeth Artist",
  "description": "The terms and conditions for using the official website of Jenny Elizabeth, artist.",
  "datePublished": "2025-04-18",
  "dateModified": "2025-04-19",
  "publisher": {
    "@type": "Organization",
    "name": "Jenny Elizabeth Artist",
    "url": "https://www.jennyelizabeth.com/"
  },
  "termsOfService": "https://www.jennyelizabeth.com/terms-and-conditions"
}
</script>`

Because the Terms of Service / Terms of Use / Terms and Conditions can include specific sections depending on the Webpage type and because these pages contain specific information site viewers need to know, I think it would be good to have an option to list each section included on a Terms page.

**Specific cases of use:** 
Terms of Service pages often cover distinct topics, such as legal language and operations policies and these sections contain details unique to the site and the user. For example, the specific sections used by a medical practitioner will be different from those used by an eCommerce site or an artist portfolio or a chat room. Also, this same schema type could be used for Privacy Policy/Cookies Policy pages or any other policy pages with distinct sections.

**Desired functionality:** 
The termsOfService type could benefit from a property such as hasSection or termSection. And this property should accept Text (for the section title), URL (if their is an anchor link), position (to denote where it is located on the list). If the sections were to be described in detail, the properties could be more structured, as with CreativeWork or WebContent. 

**Benefits of expanding ttermsOfService Schema type**
I see these as potential benefits for expanding the termsOfService type:

- Granular Understanding: It would allow search engines to understand the specific topics covered in the terms without having to parse the entire document.

- User Expectations: Site visitors will be interested in specific clauses, such as privacy, proprietary rights, permitted and prohibited uses, pricing, payment methods, return policy, conditions for user submissions, disclaimers, or dispute resolution. This could potentially lead to richer search results or more informed users.

- Clarity and Transparency: It promotes a more structured and transparent way of presenting legal terms.

**Examples**
An example of a simple termsOfService schema markup might look like this:
`{
  "@context": "https://schema.org",
  "@type": "termsOfService",
  "@id": "terms and conditions",
  "dateCreated": "2019-01-01",
  "dateModified": "2025-04-19",
  "relatedLink": "https://www.myawesomewebsite.com/privacy-policy",
  "significantLink": "https://www.copyright.gov/legislation/dmca.pdf",
  "usageInfo": "https://www.myawesomewebsite.comm/terms-and-conditions/",
  "name": [
    "Agreement and Conditions of Use",
    "Proprietary Rights",
    "Accuracy, Completeness And Timeliness",
    "Privacy Policy",
    "Products, Services & Payments",
    "Third-Parties",
    "Amendment; No Waiver",
    "Disclaimer Of Warranties; Limitation on Liability",
    "Contact Information"
  ]
}
`

If termsOfService were to mimic DefinedTerms, it might look like this:

`{
  "@context": "https://schema.org",
  "@graph": [
    {
      "@type": "termsOfService",
      "name": "Agreement and Conditions of Use",
      "position": "1",
      "url": "https://www.myawesomewebsite.com/terms-and-conditions/#comp-k9n6tp65bg"
    },
    {
      "@type": "termsOfService",
      "name": "Proprietary Rights",
      "position": "2",
      "url": "https://www.myawesomewebsite.com/terms-and-conditions/#comp-mhe4wfh7"
    },
    {
      "@type": "termsOfService",
      "name": "Accuracy, Completeness And Timeliness",
      "position": "3",
      "url": "https://www.myawesomewebsite.com/terms-and-conditions/#comp-hjz7ww5xzd0"
   }
  ]
}
`

I hope this feedback is helpful!
